platform/spec/services/temporal/activities/register_workflow_run_activity_spec.rb

```
@@ -0,0 +1,108 @@
+
+RSpec.describe Temporal::Activities::RegisterWorkflowRunActivity do
+  subject { described_class.new(context) }
+
+  let(:context) { instance_double('Temporal::Activity::Context', logger: logger) }
+  let(:logger) { instance_double('Logger', info: nil) }
+  let(:sync) { create(:sync) }
+  let(:sync_run) { create(:sync_run, sync: sync) }
+  let(:workflow_id) { "test_workflow_123" }
+  let(:run_id) { "test_run_456" }
+
+  describe '#execute' do
```

rubocop (1.68.0)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)